client/src/app/pages/applications/assessment-actions/components/assessment-actions-table.tsx

```diff
@@ -0,0 +1,95 @@
+import React from "react";
+import { useTranslation } from "react-i18next";
+import { Table, Tbody, Td, Th, Thead, Tr } from "@patternfly/react-table";
+
+import { useLocalTableControls } from "@app/hooks/table-controls";
+import {
+  ConditionalTableBody,
+  TableHeaderContentWithControls,
+  TableRowContentWithControls,
+} from "@app/components/TableControls";
+import { NoDataEmptyState } from "@app/components/NoDataEmptyState";
+import { Questionnaire } from "@app/api/models";
+import { Button } from "@patternfly/react-core";
+export interface AssessmentActionsTableProps {
+  questionnaires: Questionnaire[];
+}
+
+const AssessmentActionsTable: React.FC<AssessmentActionsTableProps> = ({
+  questionnaires,
+}) => {
+  const tableControls = useLocalTableControls({
+    idProperty: "name",
```

Comment:
One more thought: Should we use `idProperty: "id"` instead of `"name"` here? Do we need it to be a string (idProperty can refer to either a string field or a number field)? Are app names guaranteed to be unique?

Reply:
Oh sorry, I thought I was looking at the app table here, scratch that (maybe?)